code/game/machinery/computer/orders/order_computer/order_computer.dm

```diff
@@ -120,6 +120,10 @@ GLOBAL_LIST_EMPTY(order_console_products)
 				return
 			if(!purchase_items(used_id_card))
 				return
+			//So miners cant spam crates for a very low price
+			if(!check_order_cost(grocery_list))
```

Comment:
You can use ``get_total_cost`` here instead

Reply:
ngl didnt saw it